packages/google_maps_flutter/android/src/main/java/io/flutter/plugins/googlemaps/GoogleMapController.java

```diff
@@ -369,6 +371,17 @@ public boolean onMarkerClick(Marker marker) {
     return markersController.onMarkerTap(marker.getId());
   }
 
+  @Override
+  public void onMarkerDragStart(Marker marker) {}
```

Comment:
It would be great if you add support for `onMarkerDragStart` and `onMarkerDrag` as part of this PR as well.

Reply:
I could do that but I can't imagine the use case for it and thus can't add an example code for it. Moreover, the way complex types are passed between dart and native code (via JSON serialization) I don't want to introduce unnecessary CPU load when marker is being dragged.

Reply:
My use case is that I have polygons with draggable points. As the marker moves, the polygon would need to redraw. Is there some way to bypass the CPU load when onMarkerDrag isn't supplied?

Reply:
Do you absolutely need to update your poly's while dragging? I'm not sure if it's even possible.